src/components/atoms/NetworkIndicator.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react'
+import Tooltip from '@material-ui/core/Tooltip'
+import CheckCircleOutlineOutlinedIcon from '@material-ui/icons/CheckCircleOutlineOutlined'
+import ErrorIcon from '@material-ui/icons/Error'
+import WarningIcon from '@material-ui/icons/Warning'
+import { green, yellow } from '@material-ui/core/colors'
+
+export interface NetworkIndicatorProps {
+  iconClassName?: string
+  currentNetworkId?: number
+  requiredNetworkId?: number
+  onRightNetworkMessage?: string
+  onWrongNetworkMessage?: string
+  noNetworkMessage?: string
+}
+
+const defaultOnRightNetworkMessage = 'You are on the right network'
+const deaulftOnWrongNetworkMessage = 'You are on the wrong network'
+const defaultNoNetworkMessage = 'You are not connected to any network'
+
+export enum NetworkStatus {
+  NO_NETWORK = 1,
+  NETWORK_MISSMATCH = 2,
+  RIGHT_NETWORK = 3
+}
+
+const getNetworkStatus = (currentNetworkId, requiredNetworkId): NetworkStatus => {
+  if (currentNetworkId) {
+    if (currentNetworkId === requiredNetworkId) {
+      return NetworkStatus.RIGHT_NETWORK
+    }
+    return NetworkStatus.NETWORK_MISSMATCH
+  }
+  return NetworkStatus.NO_NETWORK
+}
```

Comment:
I don't like this nesting
```suggestion
const getNetworkStatus = (currentNetworkId, requiredNetworkId): NetworkStatus => {
  if (!currentNetworkId) return NetworkStatus.NO_NETWORK
  if (currentNetworkId === requiredNetworkId) return NetworkStatus.RIGHT_NETWORK
  return NetworkStatus.NETWORK_MISSMATCH
}
```

Reply:
Thank you for the suggestion, will apply this change with the new naming 🙂